test/falco_tests.yaml

```diff
@@ -1025,13 +1000,6 @@ trace_files: !mux
       - open_12: 0
       - open_13: 0
 
-  list_append_failure:
```

Comment:
Why is this removed? Same goes for `macro_append_failure` and `rule_append_failure`.

Reply:
These test cases were actually duplicates--{list,rule}_append_failure were duplicates of invalid_{list,rule}_append_dangling, just without the -V argument. macro_append_failure was a duplicate of invalid_append_macro_dangling, just without the -V argument and using a different rules file.

The point of the tests was really to check the validation, so I kept the validation versions.